frappe/custom/doctype/custom_field/custom_field.js

```diff
@@ -25,6 +25,14 @@ frappe.ui.form.on('Custom Field', {
 		frm.toggle_enable('dt', frm.doc.__islocal);
 		frm.trigger('dt');
 		frm.toggle_reqd('label', !frm.doc.fieldname);
+
+		const customize_form_link = "<a href='/app/customize-form'>Customize Form</a>";
```

Comment:
I want to go to the customize form for the specific doctype. I think it might work if we add a query param called `doctype`. Otherwise, we might need an onclick listener based on which we do frappe.set_route with route_options set.